src/Mod/Part/App/TopoShapePyImp.cpp

```diff
@@ -2037,22 +2031,6 @@ PyObject* TopoShapePy::makePerspectiveProjection(PyObject *args)
     }
 }
 
-/*!

```

Comment:
Really? Don't see this is a comment?

Reply:
I removed this :
/*!
from pivy import coin
rot=Gui.ActiveDocument.ActiveView.getCameraOrientation()
vdir=App.Vector(0,0,-1)
vdir=rot.multVec(vdir)
udir=App.Vector(0,1,0)
udir=rot.multVec(udir)
pos=Gui.ActiveDocument.ActiveView.getCameraNode().position.getValue().getValue()
pos=App.Vector(*pos)
shape=App.ActiveDocument.ActiveObject.Shape
reflect=shape.reflectLines(ViewDir=vdir, ViewPos=pos, UpDir=udir, EdgeType="Sharp", Visible=True, OnShape=False)
Part.show(reflect)
 */

Reply:
It's a comment for doxygen about how to use this function in Python. So, it must stay.